wlauto/workloads/multiapp/__init__.py

```diff
@@ -0,0 +1,164 @@
+            googlephotos_package + '/com.google.android.apps.photos.onboarding.IntroActivity',
+            googlephotos_package + '/com.google.android.apps.photos/.share.ShareActivity',
+
+            gmail_package + '/com.google.android.gm.ComposeActivityGmail',
+
+            skype_package + '/com.skype.android.app.main.SplashActivity',
+            skype_package + '/com.skype.android.app.signin.SignInActivity',
+            skype_package + '/com.skype.android.app.signin.UnifiedLandingPageActivity',
+            skype_package + '/com.skype.raider/com.skype.android.app.contacts.PickerActivity',
+            skype_package + '/com.skype.android.app.chat.ChatActivity']
+    description = '''
+    Workload to test how responsive a device is when context switching between
+    appplication tasks. It combines workflows from googlephotos, gmail and
+    skype.
+
+    Credentials for the user account used to log into the Skype app have to be provided
+    in the agenda, as well as the display name of the contact to call.
+
+    For reliable testing, this workload requires a good and stable internet connection,
+    preferably on Wi-Fi.
+
+    Although this workload attempts to be network independent it requires a
+    network connection (ideally, wifi) to run. This is because the welcome
+    screen UI is dependent on an existing connection.
+
+    SETUP: This workload requires the Google Photos apk to be placed in the
+    dependencies directory and the Gmail and Skype apks to be placed in the
+    directory of this file.
+
+    Additionally, this workload requires a JPEG file to present in the dependencies directory.
+
+    Test description:
+     1. Two images are copied to the device
+     2. Googlephotos is started in offline access mode
+     3. The first image is selected and shared with Gmail
+     4. Enter recipient details in the To: field
+     5. Enter text in the Subject edit box
+     6. Enter text in the Compose edit box
+     7. Attach the shared image from Googlephotos to the email
+     8. Click the Send mail button
+     9. Return to Googlephotos and login to Skype via share action
+    10. Return to Googlephotos and share the second image with Skype
+    11. Select a recipient from the Contacts list
+    12. The second image is posted to recipient
+    '''
+
+    parameters = [
+        Parameter('test_image', kind=str, default='uxperf_1600x1200.jpg',
+                  description='''
+                  An image to be copied onto the device that will be shared
+                  across multiple apps
+                  '''),
+        Parameter('recipient', kind=str, mandatory=True,
+                  description='''
+                  The email address of the recipient.  Setting a void address
+                  will stop any mesage failures clogging up your device inbox
+                  '''),
+        Parameter('login_name', kind=str, mandatory=True,
```

Comment:
It's not clear what login names and passwords need to be provided here -- this is not covered in the setup instructions.